<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-G0D562V" ignored="0" name="Test" tests="8" failures="4" timestamp="27 Feb 2020 08:54:49 GMT" time="140.942" errors="0">
  <testcase name="caseEight" time="0.046" classname="Cases.CaseTwo">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Cases.CaseTwo.caseEight(CaseTwo.java:51)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- caseEight -->
  <testcase name="caseEight" time="0.005" classname="Cases.CaseTwo">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Cases.CaseTwo.caseEight(CaseTwo.java:51)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- caseEight -->
  <testcase name="caseEight" time="0.004" classname="Cases.CaseTwo">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Cases.CaseTwo.caseEight(CaseTwo.java:51)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- caseEight -->
  <testcase name="caseFive" time="0.003" classname="Cases.CaseTwo"/>
  <testcase name="caseSeven" time="0.003" classname="Cases.CaseTwo"/>
  <testcase name="caseSix" time="0.003" classname="Cases.CaseTwo">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Cases.CaseTwo.caseSix(CaseTwo.java:41)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- caseSix -->
  <testcase name="caseOne" time="0.005" classname="Cases.CaseOne"/>
  <testcase name="clickNewEvent" time="10.678" classname="Cases.CaseOne"/>
</testsuite> <!-- Test -->
